### PR TITLE
Vsebp 11 frontend display of fetched events

### PR DESCRIPTION
**Pull request by Kasper-S-Jensen**
- Display fetched in events in map view and list view
This should be cached, and clustering of markers is needed
image
image

### CODE REVIEW

**Review by Kutaiba-Nezar-Kashmar — APPROVED** [review]
🚀🚀

File: app/src/main/java/io/github/viabachelora23michaelkutaibakasper/bprapp/data/repository/EventRepository.kt [hunk 1]
```diff
@@ -2,41 +2,42 @@ package io.github.viabachelora23michaelkutaibakasper.bprapp.data.repository
...
+import io.github.viabachelora23michaelkutaibakasper.bprapp.AllPublicEventsQuery
+
 import io.github.viabachelora23michaelkutaibakasper.bprapp.data.domain.Event
+import io.github.viabachelora23michaelkutaibakasper.bprapp.data.domain.GeoLocation
+import io.github.viabachelora23michaelkutaibakasper.bprapp.data.domain.Location
 
 class EventRepository : IEventRepository {
 
-    private val URL = "https://countries.trevorblades.com/"
+    private val URL = "https://api-gateway-6tyymw4cxq-ew.a.run.app/"
```
Comment (michaelbui99) [inline app/src/main/java/io/github/viabachelora23michaelkutaibakasper/bprapp/data/repository/EventRepository.kt:13]: Are there any way to make this more configurable e.g. through some config file or do we just have to edit the repository when the URL changes?

**Review by michaelbui99 — APPROVED** [review]
🚀🚀🚀

File: app/src/main/java/io/github/viabachelora23michaelkutaibakasper/bprapp/data/repository/EventRepository.kt (hunk 1 above)
Comment (Kasper-S-Jensen) [reply app/src/main/java/io/github/viabachelora23michaelkutaibakasper/bprapp/data/repository/EventRepository.kt:13]: Good question. We should look into that

Comment (Kutaiba-Nezar-Kashmar) [reply app/src/main/java/io/github/viabachelora23michaelkutaibakasper/bprapp/data/repository/EventRepository.kt:13]: You can add them to the manifest, its like appsetting.json in .net

Comment (Kasper-S-Jensen) [reply app/src/main/java/io/github/viabachelora23michaelkutaibakasper/bprapp/data/repository/EventRepository.kt:13]: Ahh, that could work!

**Pull request merged**